### ISSUE TITLE
rating(Brendan) search doesn't work; neither does sort-by-star

### ISSUE THREAD

**Issue by brendano** [plasma-umass/Apply2#87]
I was trying to bookmark records using the star, but that sort column doesn't work.  Is there a way to filter to them?

also, i tried giving them all ratings then rating(Brendan) >= 3, but that doesn't work and returs 0 records.

**Comment by brendano** [comment]
don't worry about fixing it, i figured out how to do what i want (i just make my list into a query disjunction)

**Comment by arjunguha** [comment]
It turns out that `rating(Bren) > 3` works, but `rating(Brendan) > 3` does not...

**Comment by arjunguha** [comment]
Oh crap. It is searching by email.

`rating(brenoco@gmail.com) > 3` works.

I'll fix this.

**Comment by mcorner** [comment]
I just got bitten by whatever this is:

rating(Grupen) > 0 works
rating(Rod) > 0 doesn’t

rating(Saha) > 0 doesn’t
rating(Barna) > 0 works

rating(Emery) > 0 works
rating(Berger) > 0 works

**Comment by arjunguha** [comment]
Right. That's b/c the thing in parens needs to be a fragment of their email address, not their name.
Apparently, Barna's email address doesn't contain Saha.
